front/src/components/device/ShutterButtons.css

```
@@ -0,0 +1,31 @@
+
+  /* Visually hidden, not display: none — the label keeps naming the button
+     for screen readers while the icon takes its place on screen. */
+  .shutter-btn-label {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    padding: 0;
+    border: 0;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)